torch/_export/__init__.py

```diff
@@ -235,7 +236,9 @@ def export(
             )
 
             if _add_runtime_assertions:
-                exported_program = exported_program._add_runtime_assertions()
+                exported_program = exported_program._add_runtime_assertions(
+                    functionalize_assertions=_functionalize_runtime_assertions,
```

Comment:
can shorten `functionalize_assertions` arg to simply `functionalize` (because we already call `_add_runtime_assertions`)